### PR TITLE
Remove colibri renaming

### PR DESCRIPTION
**Pull request by angolini**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 213123b2590f7183674e071bd5e75f7e43053345 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1110/docs/artifacts/html/index.html

File: source/_static/csv/supported-boards.csv [hunk 1]
```diff
@@ -5,16 +5,15 @@ Arm Cortex A5 DesignStart,a5ds
...
+:ref:`NXP i.MX6 ULL EVK with secure boot enabled <ref-rm_board_imx6ullevk>`,imx6ullevk-sec
 EA iMX7ULP-UCOM,imx7ulpea-ucom
-:ref:`NXP iMX8M-MINILPD4 EVK <ref-rm_board_imx8mmevk>`,imx8mmevk
-:ref:`NXP iMX8M-MINILPD4 EVK with secure boot enabled <ref-rm_board_imx8mmevk>`,imx8mm-lpddr4-evk-sec
-:ref:`NXP iMX8MQuad EVK <ref-rm_board_imx8mqevk>`,imx8mqevk
-:ref:`NXP Toradex Apalis-iMX6 <ref-rm_boards_apalis-imx6>`,apalis-imx6
-:ref:`NXP Toradex Apalis-iMX6 with secure boot enabled <ref-rm_boards_apalis-imx6>`,apalis-imx6-sec
-NXP Toradex Apalis-iMX8QM,apalis-imx8
-NXP Toradex Colibri-iMX7 (Aster),colibri-imx7-emmc
+:ref:`NXP i.MX 8M Mini EVK <ref-rm_board_imx8mmevk>`,imx8mmevk
```
Comment (ricardosalveti) [inline source/_static/csv/supported-boards.csv:11]: Should we also update the machine name with the new syntax (hardknott)?

**Comment by doanac** [comment]
Docs for 67bdf703189f859e426a30ed7da2c9cd56d7a2d5 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1113/docs/artifacts/html/index.html

File: source/_static/csv/supported-boards.csv (hunk 1 above)
Comment (angolini) [reply source/_static/csv/supported-boards.csv]: Should I change also the title, references and filenames? I'm not sure what is the consequence of changing that, I mean, the redirects and so on. What do you think @ricardosalveti ?

Comment (ricardosalveti) [reply source/_static/csv/supported-boards.csv]: Yeah, let's avoid the title for now to avoid breaking links.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Review by vanmaegima — APPROVED** [review]

**Pull request merged**